sdk/servicebus/service-bus/README.md

```diff
@@ -66,7 +66,16 @@ You can instantiate this class using its constructors:
 
 ### Key concepts
 
-Once you have initialized the [ServiceBusClient][sbclient] class you will be able to:
+Once you've initialized a `ServiceBusClient`, you can interact with these resources within a
+Service Bus Namespace:
```

Comment:
If the intention is to first talk about the service side concepts (queues, topics, subscriptions) and then the client side concepts (sender, receiver, session receiver), then I don't understand why we need to start this with "Once you've initialized a `ServiceBusClient`"